.github/workflows/verify.yml

```diff
@@ -16,10 +17,18 @@ jobs:
     runs-on: 'ubuntu-latest'
     steps:
     - uses: actions/checkout@v4
+
     - name: "Setup Build Environment"
       uses: ./.github/actions/setup-environment
 
+    - name: 'Setup Gradle'
+      uses: gradle/actions/setup-gradle@v3
+      with:
```

Comment:
In the setup action the key is used to get the access token, which is then actually used. In the build action it's there more for the remote cache push, since this is an OSS project. I can also remove it from the build action and as a check for remote cache push in the `settings.gradle.kts`.